### PR TITLE
Added badges using travis and other services

### PR DESCRIPTION
**Pull request by syedmohsinbukhari**
### Change Log
----------------------------------------
- Added badges for build passing and code coverage

### CODE REVIEW

**Comment by Bazarovay** [comment]
Sexy!

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
```
Comment (Bazarovay) [inline .coveralls.yml:1]: What is this?

Comment (syedmohsinbukhari) [reply .coveralls.yml:1]: this is for the code coverage badge

Comment (syedmohsinbukhari) [reply .coveralls.yml:1]: need to use coveralls.io for this

**Comment by Bazarovay** [comment]
Travis error. Please fix.
image

**Comment by Bazarovay** [comment]
Addresses #7

**Review by Bazarovay — CHANGES_REQUESTED** [review]
Badge is not appearing

**Review by Bazarovay — APPROVED** [review]

**Comment by Bazarovay** [comment]
Merging. This is a great PR

**Pull request merged**